.circleci/config.yml

```diff
@@ -42,6 +44,8 @@ jobs:
         command: make -s test
         name: ddev tests
         no_output_timeout: "40m"
+    - store_test_results:
```

Comment:
This needs to point to the full path of where the gotestsum command placing the XML. Or we can add a step after make test to mv/cp the file over